xcode/Subconscious/Shared/Components/Common/Byline/PetnameView.swift

```diff
@@ -71,6 +71,7 @@ struct PetnameView: View {
             return aliases.filter { alias in
                 name != alias.leaf
             }
+            .uniquing(with: { $0 })
```

Comment:
Fixes an edgecase, also maybe a bit too clever.

Reply:
Nit: I prefer the `.uniquing(with: { value in value })` to reduce the amount of magical syntax newbies have to learn.

Alternatively, `.uniquing(with: \.self)` since we use keypaths elsewhere. Keypaths are another Swiftism, but a more useful one, since they do more than sugar (type-safe dynamic property lookup).